client/extensions/woocommerce/app/promotions/fields/form-field.js

```diff
@@ -52,12 +52,16 @@ const FormField = ( {
 		{ 'fields__fieldset-children-enableable': enableCheckbox }
 	);
 
+	const formLabel = ( enableCheckbox || labelText ) && (
+		<FormLabel id={ fieldName + '-label' } required={ isRequired }>
+			{ enableCheckbox }
+			{ labelText }
+		</FormLabel>
+	);
+
 	return (
 		<FormFieldset className="fields__fieldset">
-			<FormLabel id={ fieldName + '-label' } required={ isRequired }>
```

Comment:
@ryelle is removing this FormLabel going to mess up accessibility? If so, how should we have it there for a11y but not visible?

Reply:
You can wrap it inside [`<ScreenReaderText>`](https://wpcalypso.wordpress.com/devdocs/design/screen-reader-text), but a better solution would probably be to add an `aria-label` to the un-labelled selects directly.

Reply:
I've got a note to follow up on this after the appliesTo component gets updated.